packages/data-broker/src/DataBroker/DataBroker.ts

```diff
@@ -383,56 +401,6 @@ export class DataBroker {
     });
   };
 
-  private mergeAnalytics = (
```

Comment:
Analytics merging is kind of complicated. Here I just extracted the existing `DataBroker.mergeAnalytics` method into a pure function.

`DataBroker` has a lot of responsibilities and more have been added over the years... Good old separate methods/classes per concern can help tame complexity